packages/frontend-api/src/Resources/config/graphql-types/ModelType/Complaint/ComplaintItemDecorator.types.yaml

```diff
@@ -4,8 +4,14 @@ ComplaintItemDecorator:
     config:
         fields:
             orderItem:
-                type: "OrderItem!"
+                type: "OrderItem"
                 description: "Order item"
+            productName:
```

Comment:
should not be there the product itself in the schema as well? (so the product image, or the product link detail can be presented if possible)

Reply:
This change isn't required for anything on the frontend in this ticket. I'd prefer to add it in a follow-up ticket (complaints list and detail) where it can be utilized.

Reply:
ok, fair enough